auslib/admin/views/releases.py

```diff
@@ -116,7 +118,9 @@ def changeRelease(release, changed_by, transaction, existsCallback, commitCallba
                     cef_event("Bad input", CEF_WARN, errors=msg, release=rel)
                     return Response(status=400, response=msg)
                 if 'hashFunction' in releaseInfo['data'] and hashFunction and hashFunction != releaseInfo['data']['hashFunction']:
-                    msg = "hashFunction '%s' doesn't match the one on the release object ('%s') for release '%s'" % (hashFunction, releaseInfo['data']['hashFunction'], rel)
+                    msg = "hashFunction '%s' doesn't match the one on the release object ('%s') for release '%s'".format(
+                        hashFunction, releaseInfo["data"]["hashFunction"],
+                    )
                     cef_event("Bad input", CEF_WARN, errors=msg, release=rel)
```

Comment:
Whoops, I didn't realize I'd removed that. cef_event won't fill in log messages AFAIK, so I need to fix up the formatting (including to switch %s to {} style).